docs/development/tools.md

```diff
@@ -0,0 +1,106 @@
+# Development Tools
+
+## Intro
+
+This project uses a set of developer tools to ensure code quality, maintainability, and consistency.
+These tools are installed from `ci/tools` using go install.
+
+In order to install the tools, run:
+```bash
+  make install-tools
+```
+
+## Tools
+
+## Code Quality & Static Analysis
```

Comment:
I am wondering if these tools (most of them) are redundant when using golangci?

Reply:
some are, indeed. I'm just photographing the current status. When introducing the golangci configuration, we will remove tools that are not needed anymore.
Good catch.